packages/utilities/fast-benchmarks/README.md

```diff
@@ -0,0 +1,27 @@
+# FAST Benchmarks
+This is small utility library for creating and running benchmark tests for FAST scenarios.
+
+## Usage
+## Adding a Benchmark
+To add a benchmark, create a new directory in `/benchmarks/` to hold your benchmark, eg `/benchmark/my-benchmark/`. Next, create an `index.ts` file and optionally an `index.html` file.
+
+### index.ts
+This file sets up the benchmark scenario and should export an instance of `Benchmark` as the **default export**:
+
+```ts
+export default new Benchmark(() => /* do something */);
+```
+
+> Benchmark is in the global scope, so there is no need to import it.
+
+### index.html
```

Comment:
Since "Adding a Benchmark" is under "Usage" I think this should have 3 `###` and any sub section should also have an additional `#` to indicate hierarchy.

```suggestion
### Adding a Benchmark
To add a benchmark, create a new directory in `/benchmarks/` to hold your benchmark, eg `/benchmark/my-benchmark/`. Next, create an `index.ts` file and optionally an `index.html` file.

#### index.ts
This file sets up the benchmark scenario and should export an instance of `Benchmark` as the **default export**:

```ts
export default new Benchmark(() => /* do something */);
```

> Benchmark is in the global scope, so there is no need to import it.

#### index.html
```